README.md

```diff
@@ -53,6 +53,46 @@ without SSL there are limited options for ensuring the privacy of the
 plaintext transport. See the section on [Platform authentication
 support](#platform-winrm-authentication-support).
 
+If your cloud provides you with an rdp ssl fingerprint, you can
```

Comment:
I'd prefer to keep the instructions very minimal. Just a couple sentences stating if you know of a valid fingerprint that exists on the node, specify that as the `--ssl_peer_fingerprint` to be validated. This must reside in the certificate store on the node and there must be an active HTTPS winrm listener created with that certificate.

My reasoning here is:
* the readme is already quite lengthy
* it includes instructions on setting up SSL
* For testing purposes, now that both linux and windows workstations can use NTLM encryption over HTTP, thats the easiest route people are likely to pursue.

Does that sound reasonable?